scripts/Synapse/convertParquetMPI.ipynb

```diff
@@ -59,9 +68,25 @@
         "snapshot_bucket_name = \"bundle-snapshots\"\n",
         "file_system_name = \"patient-data\"\n",
         "file_path = \"MPI.parquet\"\n",
-        "storage_account_url = f\"https://{account_name}.blob.core.windows.net\"\n",
+        "storage_account_url = f\"https://{account_name}.blob.core.windows.net/\"\n",
         "mpi_incoming_file_path = f\"abfss://{file_system_name}@{account_name}.dfs.core.windows.net/{file_path}\"\n",
         "\n",
+        "# Set up for writing to blob storage\n",
+        "blob_relative_path = \"\"\n",
+        "linked_service_name = \"AzureBlobStorage1\" \n",
```

Comment:
This doesn't look like the correct name for a link service covered by Terraform.